app/services/total_rewards_update_service.rb

```diff
@@ -14,7 +14,7 @@ def call
     if completed_epoch
       vote_rewards = total_rewards_from_vote_accounts(completed_epoch)
       stake_rewards = total_rewards_from_stake_accounts(completed_epoch)
-      if vote_rewards > 0 && stake_rewards > 0
+      if (vote_rewards > 0 && stake_rewards > 0) || @network == "pythnet"
```

Comment:
What about removing `if (vote_rewards > 0 && stake_rewards > 0) || @network == "pythnet"` condition and just making an update every time?

Reply:
on mainnet if the rewards are 0 that means someting is wrong (like bad rpc call) and it should never be 0